webapp/pages/index.vue

```diff
@@ -92,9 +92,39 @@ export default {
       ],
     }
   },
+
+  mounted() {
+    if (this.hashtag) {
+      this.changeFilterBubble({ tags_some: { name: this.hashtag } })
+    }
+    if (!this.hasAgreedToLatestTermsAndConditions) {
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-08-16T08:48:41Z" title="Friday, August 16th 2019, 10:48:41 am +02:00">Aug 16, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -92,9 +92,39 @@ export default {
       ],
     }
   },
+
+  mounted() {
+    if (this.hashtag) {
+      this.changeFilterBubble({ tags_some: { name: this.hashtag } })
+    }
+    if (!this.hasAgreedToLatestTermsAndConditions) {
```


Ideally, this should not happen client side when you mount the page component. Instead, as soon as the user object is loaded, you can redirect to another page immediately and ask for confirmation.